package/harvester-os/files/usr/sbin/harv-install

```diff
@@ -435,6 +435,14 @@ update_grub_settings()
     if ! [ -f ${TARGET_FILE} ]; then
         touch ${TARGET_FILE}
     fi
+    # /etc/cos/bootargs.cfg appends a new variable $third_party_kernel_args
+    # if harvester config has os.externalStorageConfig.additionalKernelArguments specified
+    # then these will be mapped to HARVESTER_EXTERNALSTORAGECONFIG_ADDITONALKERNEL_ARGUMENTS
```

Comment:
it is correct, since the bootargs.cfg now contains an extra variable to handle this: https://github.com/harvester/harvester-installer/pull/778/files#diff-ef09eae8720e1a7b7c766630cec71409475ca7139fce60d4654ce6c20475e944R7

Reply:
Sorry, I mean the comment has it as HARVESTER_EXTERNALSTORAGECONFIG_ADDITONALKERNEL_ARGUMENTS whereas the code below uses HARVESTER_ADDITIONAL_KERNEL_ARGUMENTS